versioned_docs/version-22.10/connect/openid.md

```diff
@@ -39,7 +37,7 @@ Configure Identity Provider information:
 - **Scopes**: defines the scopes of the identity provider, for example `openid`. Separate scopes by spaces.
   > Depending on the identity provider, it is necessary to enter several scopes in order to retrieve the claim which will
   > identify users. This is indicated in the provider's configuration documentation.
-  - **Login claim value**: defines which of the variables returned by **Introspection Token Endpoint** or **User Information Endpoint**
+  - **Login attribute path**: defines which of the variables returned by **Introspection Token Endpoint** or **User Information Endpoint**
```

Comment:
![image](https://user-images.githubusercontent.com/78470741/195635440-4243e7bc-d074-4db9-a059-5547b15fe3b3.png)

Reply:
I think the login attibute path should not be in the yellow part